spec/system/purchase_system_spec.rb

```diff
@@ -151,26 +154,24 @@
         it "Does not include inactive items in the line item fields" do
           visit url_prefix + "/purchases/new"
 
-          item = Item.alphabetized.first
+          select @storage_location.name, from: "purchase_storage_location_id"
+          expect(page).to have_content(@item.name)
+          select @item.name, from: "purchase_line_items_attributes_0_item_id"
```

Comment:
Same. In general I'm confused here why some tests are removing instance variables and others are adding them.

Reply:
Basically I wanted to make minimal changes. A lot of the order dependence issues stemmed from needing to set `organization:` on things. Thus, where possible I set it with the instance variable `organization` but since the db is not getting reset within a spec file some of the specs modify `@organization` and cause other ones to fail.

In general I could remove all the changes to `@organization` with a created organization but I think that will expand the PR a decent amount.

EX: with some of the shared examples, I have to change all the parts that use it if I move off instance variables.

Reply:
I basically went:
- does this work? If yes leave
- does this work if I add instance variables to it? If yes add them
- otherwise replace instance variables with created ones.